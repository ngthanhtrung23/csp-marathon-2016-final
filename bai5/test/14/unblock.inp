..1222
3.1455
30046.
.7..68
.79968
.AABB.
