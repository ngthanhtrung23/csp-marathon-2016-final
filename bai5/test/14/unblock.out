23
3 U 1
0 L 1
1 D 2
2 L 1
4 D 1
5 L 2
6 U 1
8 U 3
9 R 2
1 D 1
4 D 1
0 R 2
7 U 3
0 L 2
1 U 1
4 U 1
9 L 4
1 D 1
4 D 1
A L 1
B L 1
6 D 2
0 R 4

